new-log-viewer/src/services/LogFileManager.ts

```diff
@@ -1,9 +1,12 @@
+/* eslint-disable max-lines */
```

Comment:
If we're really short of time, let's consider changing the limit instead of completely disabling the rule:
```
/* eslint max-lines: ["error", 400] */
```